content/3_Silver/Prefix_Sums_2.mdx

```diff
@@ -102,11 +102,94 @@ export const problems = {
 
 <FocusProblem problem={problems.maxsum} />
 
+### Solution
+
+<Spoiler title="Solution">
+
+Consider the prefix sum array. The maximum subarray sum is the maximum possible $p[i]-p[j]$ where $j\leq i$. We can iterate over each index $i$ and keep a running minimum $p[j]$ value. This yields the maximum subarray sum for each possible right bound, and the maximum over all these values is our answer.
+
+<LanguageSection>
+
+<CPPSection>
+
+C++ code:
+
+```cpp
+#include <bits/stdc++.h>
+
+using namespace std;
+using ll = long long;
+using vl = vector<ll>;
+
+int main(){
+	int n; cin >> n;
+	vl dp(n+1, 0); // Prefix sum array initially filled with 0's
+	for(int i=1; i<=n; i++){
+		ll x; cin >> x;
+		dp[i] = dp[i-1] + x; // Compute the prefix sum at each element
+	}
+	ll max_subarray_sum = dp[0], min_prefix_sum = dp[0];
+	for(int i=1; i<=n; i++){
+			// Max subarray sum is the maximum difference between two prefix sums
```

Comment:
I thought the standard was to have comments be on the same indentation as the code?

Reply:
Looks fine tho